Zend/zend_atomic.h

```diff
@@ -82,7 +76,7 @@ static zend_always_inline bool zend_atomic_bool_exchange_ex(zend_atomic_bool *ob
 }
 
 static zend_always_inline bool zend_atomic_bool_load_ex(const zend_atomic_bool *obj) {
-	return __c11_atomic_load(&obj->value, __ATOMIC_SEQ_CST);
+	return __c11_atomic_load(&((zend_atomic_bool *) obj)->value, __ATOMIC_SEQ_CST);
```

Comment:
If we need to cast the const-ness away, why do we declare the argument as const in the first place?

Reply:
FYI, prior to this PR it wasn't declared as const on this platform.

Reply:
I think `load` should use const pointer: it won't change data that the pointer pointed to, and this is a clang specific behavior, this is fixed since clang 8.0:

https://github.com/llvm/llvm-project/commit/b4b1f59869b1045258787f5a138f9710859cfe95

Will PHP drop support of clang 7 and older?

Reply:
It's just a workaround, and it should be const.
As some atomic implementations use `fetch_add(0)` to implement `load()`, the parameter type of `load()` is still const to indicate that the value will not be changed.